recipes/debugging/vs-code-debug.md

```diff
@@ -1,40 +1,57 @@
-# VS Code Amber Debug
+# Debugging Crystal Projects using VSCode
 
-Debugging an Amber project on VSCode with GDB and [Native Debug](https://github.com/crystal-lang-tools/vscode-crystal-lang/wiki/Useful-extensions#debugging).
+This tutorial have some tips and tricks to debug Crystal projects with tools like GDB or LLDB using debugger clients like [Native Debug](https://marketplace.visualstudio.com/items?itemName=webfreak.debug) for VSCode.
 
-## Requisites
+## Prerequisites
 
-* Crystal
-* Amber Project - See installation guide [here](/getting-started/Installation/README.md)
-* GNU Debugger (GDB) - MacOS install `brew install gdb`
-* VSCode + [Crystal Support](https://marketplace.visualstudio.com/items?itemName=faustinoaq.crystal-lang) + [Native Debug](https://github.com/WebFreak001/code-debug)
+* Crystal - See instalation guide [here](https://crystal-lang.org/docs/installation/)
+* Crystal project - See installation guide [here](https://crystal-lang.org/docs/using_the_compiler/#creating-a-project-or-library) or [here (Amber)](/getting-started/Installation/README.md)
+* VSCode with [Crystal Lang](https://marketplace.visualstudio.com/items?itemName=faustinoaq.crystal-lang) and [Native Debug](https://marketplace.visualstudio.com/items?itemName=webfreak.debug) extensions
+* GNU debugger (GDB) or LLVM debugger (LLDB) - See installation guide below
 
-> Note: Before continuing setting up the debugger make sure you have the above requisites install. These settings have been verified for a MacOS enviroment.
+To install GDB or LLDB according with your OS use one of the following commands:
+
+##### MacOS
+
+`brew install gdb lldb`
+
+##### Ubuntu
+
+`apt install gdb lldb`
+
+##### Fedora
+
+`dnf install gdb lldb`
+
+##### ArchLinux
+
+`pacman -S gdb lldb`
+
+
+> Note: Before continuing setting up the debugger make sure you have the above prerequisites installed. These settings have been verified for a MacOS and Linux's enviroments
 
 ## Steps
 
-**1. `task.json` to compile the main file with debug support**
+> By convention project directory name is equal to your application name, if you have changed it, please update `${workspaceFolderBasename}` by your prefered name configured inside `shards.yml`
```

Comment:
By convention the project directory name is the same as your application name, if you have changed it, please update `${workspaceFolderBasename}` with the name configured inside `shards.yml`